kolibri/plugins/facility/assets/src/views/FacilityConfigPage/index.vue

```diff
@@ -55,13 +55,19 @@
                 :key="setting"
                 :label="$tr('learnerNeedPasswordToLogin')"
                 :checked="!settings['learner_can_login_with_no_password']"
-                @change="toggleSetting('learner_can_login_with_no_password')"
+                @change="() => {
+                  toggleSetting('learner_can_login_with_no_password')
+                  settings['learner_can_edit_password'] &&
+                    !settings['learner_can_login_with_no_password'] ?
+                      toggleSetting('learner_can_edit_password') : null
+                }"
```

Comment:
let's move this to be a component method in the `script` section so we don't pack multiple lines of JS into a `template` attribute

Reply:
@indirectlylit thanks for the feedback! I think the backend validation is mentioned here #7281, I will take a look at it in a separate pr.